test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -611,28 +614,176 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
             },
             fnArtifact: {
               handler: 'index.handler',
-              artifact: 'artifact-function.zip',
+              package: {
+                artifact: 'artifact-function.zip',
+              },
             },
           },
         },
       });
     });
-    it('should support `package.artifact`', () => {
+
+    it.skip('should support `package.artifact`', () => {
       // Confirm that file pointed at `package.artifact` is configured as service level artifact
       //
       // Replace
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L227-L235
     });
 
-    it('should ignore `package.artifact` if `functions[].package.individually', () => {
+    it.skip('should ignore `package.artifact` if `functions[].package.individually', () => {
       // Confirm that fnIndividual was packaged independently
       //
       // Replace
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L262-L287
     });
 
-    it('should support `functions[].package.artifact`', () => {
+    it.skip('should support `functions[].package.artifact`', () => {
       // Confirm that file pointed at `functions.fnArtifact.package.artifact` is configured as function level artifact
     });
+
+    describe('with absolute artifact path', () => {
+      let absoluteArtifactFilePath;
+      let zipContent;
+
+      before(async () => {
+        const zip = new JsZip();
+        zip.file('index.js', '');
+        zipContent = await zip.generateAsync({ type: 'base64' });
+        absoluteArtifactFilePath = path.join(process.cwd(), 'newArtifact.zip');
+        await fs.promises.writeFile(absoluteArtifactFilePath, zipContent);
```

Comment:
Maybe we can reuse artifact directly from a fixture (we may eventually copy it to a new path, assuming that we rely on filename to confirm that things works)